### DISCUSSION TITLE
Frequency Error Indicator for SX126x modules

### DISCUSSION THREAD

**Discussion by IZ7BOJ** [jgromes/RadioLib#741] [General]
Hi all,
I see that a frequency error indicator is implemented for sx126x modules. 
The register address seems to be "0x076B" although I can't find any reference in the Semtech official datasheet (I'm searching into the sx1268 datasheet v1.1 , but I think it's the same for the others)
What am I missing?
Thanks in advance for the clarification
Regards,
Alfredo Vania, Ham Radio Station IZ7BOJ

**Comment by jgromes** [comment]
As the note in the documentation says:

```
WARNING: This functionality is based on SX128x implementation and not documented on SX126x.
While it seems to be working, it should be used with caution!
```

Using the SX128x frequency error reading, I was able to read it from SX126x device. Why is this working, why was it not documetned and what are the side effects (if any) are all great questions - to Semtech. From what I saw on their LoRa developer forums, a new release of the SX126x documentation is being prepared, so I'm hoping some new information will come to light.

**Reply by IZ7BOJ** [reply]
Hi Jan, thank you for the clarification. I'm testing this function on a sx1268 ant it seems to work properly. We'll wait for the next issue of the Datasheet to see if the include this register.
Regards,
Alfredo
